engine/app/components/citizens_advice_components/icons/arrow_down.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= render CitizensAdviceComponents::Icons::Base.new(size: size, icon_name: self.class.name) do %>
```

Comment:
Having to pass in the class name in every icon component isn't the DRY-est, but wasn't sure of a way of getting this from inside the base class.  I did have

```
def human_class_name
  self.class.name.....
end
```

But that's always just `base`, since I'm always creating a `Base` component with the `do` block.

Reply:
No reason the class couldn't reasonably define an `icon_name` method.

```
class ArrowLeft < CitizensAdviceComponents::Icons::Base
   def icon_name
      "arrow-left"
   end
end
```

Also just wondering if having the component inherit from `CitizensAdviceComponents::Icons::Base` but then also rendering `CitizensAdviceComponents::Icons::Base` is a little confusing here. Wondering if there's something slightly back to front here.